packages/cli/src/index.ts

```diff
@@ -30,17 +30,20 @@ const handleError = (err: Error) => {
   } else {
     // Some error messages (esp. custom ones) might have `.` at the end already.
     const message = err.message.replace(/\.$/, '');
-    logger.error(
-      `${message}. ${chalk.dim(
+    logger.error(`${message}.`);
+  }
+  if (err.stack) {
+    logger.log(err.stack);
```

Comment:
Are there cases when `err.stack` is missing? Just curious

Reply:
Oh, I see that's an old code anyway

Reply:
yea, a custom error can delete it